org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/persistence/DocumentResource.xtend

```diff
@@ -67,7 +69,15 @@ class DocumentResource {
 
 	// currently dummy implementation to get user from header authorization
 	private def String getUserName(HttpHeaders headers) {
-		return headers.getHeaderString('Authorization').split(':').head
+		val authorization = headers.getHeaderString('Authorization')
+		var credential = ""
+		if (authorization.startsWith("Basic")) {
+			val base64Credential = authorization.replaceFirst("^Basic ", "")
+			credential = new String(Base64.decoder.decode(base64Credential), Charset.forName("UTF-8"))
```

Comment:
I would prefer directly returning the split result here instead of assigning it to credential

Reply:
Use `StandardCharsets.UTF_8` instead of `Charset.forName("UTF-8")`